Project.toml

```
@@ -14,7 +14,7 @@ UUIDs = "cf7118a7-6976-5b1a-9a39-7adc72f591a4"
 [compat]
 AWS = "1"
 MbedTLS = "1"
-Memento = "1"
+Memento = "~1.0, ~1.1, ~1.2"
```

Could this be simplified to?

```suggestion
Memento = "1 - 1.2"
```

---

No, since hyphens require Julia 1.4